src/splitInteger.test.js

```diff
@@ -1,17 +1,22 @@
+'use strict';
+/* eslint-disable no-unused-vars */
+/* eslint-disable max-len */
+
 const splitInteger = require('./splitInteger');
 
 test(`should split a number into equal parts if a value is divisible by a numberOfParts`, () => {
-
+  expect(splitInteger(8, 2)).toContain(4, 4);
 });
```

Comment:
I would suggest using `toEqual` here, it would be more suitable for this comparasion